### PR TITLE
add  igridsome-starter-algrid

### PR DESCRIPTION
**Pull request by gollowars**
I created a simple starter using Algolia and infinite scroll.

### CODE REVIEW

**Review by 3imed-jaberi — COMMENTED** [review]
sgtm

File: starters/starters.yaml [hunk 1]
```diff
@@ -421,4 +421,13 @@
   preview: https://akarui.netlify.app
   author: arturoalviar
   tags: gridsome, starter, sanity, blog, portfolio, tailwind, tailwindcss, dark theme, responsive
-  platforms: sanity
\ No newline at end of file
+  platforms: sanity
+
+- title: Gridsome Algrid
+  description: Algrid is a simple starter for creating blogs using Gridsome.
```
Comment (thetre97) [inline starters/starters.yaml:427]: Could you perhaps update this description to include the blog features, like using Algolia and implementing infinite scroll?

**Review by thetre97 — CHANGES_REQUESTED** [review]

File: starters/starters.yaml (hunk 1 above)
Comment (gollowars) [reply starters/starters.yaml]: @thetre97 I've done to update the description! Could you check please?

File: starters/starters.yaml [hunk 2]
```diff
@@ -439,6 +439,15 @@
   tags: gridsome, starter, sanity, blog, portfolio, tailwind, tailwindcss, dark theme, responsive
   platforms: sanity
 
+- title: Gridsome Algrid
+  description: Algrid is a simple starter for creating blogs using Algolia and implementing infinite scroll.
```
Comment (thetre97) [inline starters/starters.yaml:443]: Sorry, can I be picky and get you to add `Algolia for search`? 🙂

Comment (gollowars) [reply starters/starters.yaml:443]: @thetre97 
OK! Thank you for your feedback i'm poor of english.. sorry.

Is this alright with you?
`description: Algrid is a simple starter for creating blogs using Algolia for search and implementing infinite scroll.`

Comment (thetre97) [reply starters/starters.yaml:443]: That looks great! I personally would add a comma after `search`, just to separate the two parts...

Comment (gollowars) [reply starters/starters.yaml:443]: OK! I finally understand! I'll correct it.
`description: Algrid is a simple starter for creating blogs using Algolia for search, and implementing infinite scroll.`

Comment (thetre97) [reply starters/starters.yaml:443]: Perfect 🙂

**Review by thetre97 — APPROVED** [review]

**Comment by thetre97** [comment]
Thanks for your contribution!

**Pull request merged**